base_environment.yml

```diff
@@ -12,3 +12,7 @@ dependencies:
     - pip
     - pre-commit
     - pyqt =5.12
+
+environment:
```

Comment:
This is not a `.devenv.yml` file there is no job failing, but this file is used to generate the docs.
This should not be a problem, I think conda ignores this.

Reply:
I just check devenv.
if we throw a file whitout `.devenv.yml` extension there is no include or environment handling. If some one includes this file they are handled.

Reply:
Good catch.

I changed it to `devenv.yml` for consistency and renamed to `_base_environment.devenv.yml` to signal its internal status. 👍